static/thanks.html

```diff
@@ -3,10 +3,12 @@
 <body>
 <ul>
     <li>Karl</li>
+    <li>Aras Pranckevicius</li>
```

Comment:
This is just for the patreon supporters. Could you nix this change.

Reply:
Yep, I'll rebase, I screwed up the commit (I merged an unrelated commit into my own, I believe)